sklearn/base.py

```diff
@@ -266,6 +266,7 @@ def __repr__(self):
 ###############################################################################
 class ClassifierMixin(object):
     """Mixin class for all classifiers in scikit-learn."""
+    estimator_type = "classifier"
```

Comment:
On 03/24/2015 08:49 PM, Mathieu Blondel wrote:
>
> I think the __ in __add__ is just a way to emphasize that this is a 
> special method rather than a private one. If the goal is to not mess 
> w/ tab completion I am +1 with the _ prefix but the variable should 
> really be documented as part of our public API (i.e. we won't change it).
+1
I documented it in the dev docs.